internal/config/operations.go

```diff
@@ -0,0 +1,28 @@
+package config
+
+import (
+	"github.com/layer5io/meshery-adapter-library/adapter"
+	"github.com/layer5io/meshery-adapter-library/meshes"
+)
+
+var (
+	ServiceName = "service_name"
+)
+
+func getOperations(dev adapter.Operations) adapter.Operations {
+	versions, _ := getLatestReleaseNames(3)
+
+	dev[LinkerdOperation] = &adapter.Operation{
+		Type:        int32(meshes.OpCategory_INSTALL),
+		Description: "Linkerd Service Mesh",
+		Versions:    versions,
+		Templates: []adapter.Template{
+			"templates/linkerd.yaml",
+		},
+		AdditionalProperties: map[string]string{
+			ServiceName: LinkerdOperation,
```

Comment:
Which service are we referring to here? It was intended to be the name of the Kubernetes service exposing the frontend of the application/workload, used to get the endpoint of the service and return it to the user for convenience (or maybe for testing). The operation name is most likely not the same as the name of this service.

Nevertheless, it should be documented.

Reply:
Oh I wasn't aware of this use case. In this case, it does makes sense and so can we rename this to `Workloads` or `Endpoints`? Just that I think `Services` is a generic description for this specific use.

Reply:
Yes, hmm, this use case is the `ServiceName` for which we want the endpoint (host:port) for, contrary to all the other services in the mesh an external user/service doesn't care about. Even so, a service name or id that is registered in some catalog (to generalize) and obtained by querying that catalog. Yes, agreed it should be more specific. 
`AppServiceName`, `WorkloadServiceName`,  `EndpointName`, `FrontendName`, `PublicEndpointName`, `ExposedServiceName`....